Cython/Compiler/ExprNodes.py

```diff
@@ -8052,6 +8052,10 @@ class SequenceNode(ExprNode):
     unpacked_items = None
     mult_factor = None
     slow = False  # trade speed for code size (e.g. use PyTuple_Pack())
+    needs_subexpr_cleanup = False  # set to True in code-generation if we
```

Comment:
We always need some kind of cleanup, but not always disposal/decrefs.
```suggestion
    needs_subexpr_disposal = False  # set to True in code-generation if we
```